hystrix-core/src/main/java/com/netflix/hystrix/HystrixCollapser.java

```diff
@@ -517,13 +514,24 @@ private static String getDefaultNameFromClass(@SuppressWarnings("rawtypes") Clas
         public void setResponse(ResponseType response);
 
         /**
-         * When set any client thread blocking on get() will immediately be unblocked and receive the exception.
+         * When invoked, any Observer will be OnNExted this value
+         * @param response
```

Comment:
Should this throw an IllegalStateException if called after `setResponse` or `setComplete`? That should be a developer bug I think, and not just swallowed.